test/DynamoCoreWpfTests/PublishPackageViewModelTests.cs

```diff
@@ -17,7 +18,7 @@ namespace DynamoCoreWpfTests
     public class PublishPackageViewModelTests: DynamoViewModelUnitTest
     {
 
-        [Test, Category("Failure")]
```

Comment:
this seems to be passing... this test is unrelated to this task so if it fails on CI I will just revert this.